evm/.example.hardhat.config.js

```diff
@@ -3,7 +3,7 @@ require('@nomicfoundation/hardhat-toolbox');
 
 const env = process.env.ENV || 'testnet';
 const { importNetworks, readJSON } = require('@axelar-network/axelar-contract-deployments');
-const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const chains = require(`@axelar-network/axelar-contract-deployments/axelar-chains-config/info/${env}.json`);
```

Comment:
```suggestion
const chains = require(`@axelar-network/axelar-chains-config/info/${env}.json`);
```
won't it be it's own package?

Reply:
that was my mistake by using a replace-all tool. since the user clones the whole repo to setup this, we can use the local path.

Reply:
this is for the example config though (meant to be used by other repos). the actual hardhat config of this repo can refer to the local version, yeah

Reply:
i see, so it makes sense to refer to npm package then.